### PR TITLE
Revamped service header

### PR DESCRIPTION
**Pull request by maceq687**

### CODE REVIEW

File: _layouts/services.html [hunk 1]
```diff
@@ -61,21 +61,31 @@ <h1><a href="/revops#services">RevOps services</a></h1>
 
   .header-overlay,
   .header-image {
-    height: 300px;
-    width: calc(100vw - 115px);
-    max-width: 1200px;
+    height: 16em;
```
Comment (hypesystem) [inline _layouts/services.html]: Height could also be based on page width or height by using `vh` or `vw` units.

**Review by hypesystem — APPROVED** [review]
Y E S! Very nice!

I love it, instant approved.

If you get rid of the blue header bar and make the deranged logotext and the menu icon black on the light background, I think it will work even better, feel even more free/unconstrained!

Here's what I mean:

image

Getting there will obviously take some more code changes (I just did a bit of experimentation in the browser to get here), but I think the result is good and leaves the right things in foucs: the page heading, the page tagline (which now feel connected). And less emphasis on the deranged name and menu button.

**Comment by hypesystem** [comment]
Continuing some riffing on the design, if we alternate the derangedblue and grey background we get something like this:

Screen Shot 2024-09-18 at 11 28 42

On the image I used these CSS properties to make it blend into the blue background and appear clearly as decoration and non-essential:

```css
mix-blend-mode: color-dodge;
filter: saturate(0.7);
```

Swap the mix-blend-mode to `color-burn` on the light grey background and it works there too.

If I were to continue here, I'd probably try to make the transition from grey to blue background less stark. We have a bunch of examples of how to make it less of a hard line across on the frontpage. E.g. make it a tirangle or similar.

**Comment by maceq687** [comment]
Current services header section preview:
header_redo
The changes regarding the rest of the page content deserve a separate PR

File: _layouts/services.html (hunk 1 above)
Comment (maceq687) [reply _layouts/services.html]: I'm aware, but actually in this case it works pretty well with em since it follows the h1 size

**Comment by hypesystem** [comment]
@maceq687 I'm not a huge fan of the green tint — or any non-brand color tint for that matter. Because our brand is so loose, a font and a color scheme (but lots allowed within that), diverging from it will make it really confusing and seem like a different site/brand.

I'm also not sure the contrast of the text in the header is good enough with the tint.

If you tint it background color Grey both those issues disappear.

Other than that, it looks good and can be merged.

**Comment by maceq687** [comment]
Green tint in Pipedrive (and Orange in HubSpot) services were suppose to be a sutle non towards the each of the tools brands colors. I'm not super attached to it, but kinda wanted to distinguish them somehow. I agree that the low contrast might be an issue here. Perhaps coloring the h1 instead could be a way to keep it?

**Comment by hypesystem** [comment]
Using a logo image is a much better solution than using their brand colors!

Prefer single-color logos on transparent background and make them plenty big

**Comment by hypesystem** [comment]
(I get the intent with brand colors but mixing too many colors usually just ends up confusing users. It needs to be really thorough and e.g. a whole subset of pages use a different brand color to seem intentional... or generally have a more colorful brand than we have!)

**Comment by maceq687** [comment]
Replaced custom tints with logos in headers for piepedrive and hubspot services:
hub
pipe

**Review by hypesystem — APPROVED** [review]
This looks great!

File: _layouts/services.html [hunk 2]
```diff
@@ -184,6 +193,11 @@
       margin-right: 1em;
       margin-left: 1em;
     }
+    img.service-logo {
+      height: 4em;
```
Comment (hypesystem) [inline _layouts/services.html:198]: If you want to be more specific with logo size, set width and height and specify `object-fit: contain;`

**Pull request merged**